core/protos/o2control.proto

```diff
@@ -176,7 +176,7 @@ message GetEnvironmentsReply {
 }
 message EnvironmentInfo {
     string id = 1;
-    string createdWhen = 2;
+    int64 createdWhen = 2;
```

Comment:
@graduta please note the backwards-incompatible change.

Reply:
@teo  thank you for the notification. What is the expected delivery date for this?

Reply:
@graduta this week's release. The only mandatory change for you is that `createdWhen` is now an int64 with nanoseconds since UNIX epoch. 3 additional fields were added but you're not forced to adapt to them immediately.